NCEP/gdas_utility.py

```diff
@@ -0,0 +1,264 @@
+import os
```

Comment:
Would you please add a short descript in the beginning of this file:

#Description:
#Author: your email
#Revision history: 
#      20231010:  your_name   initial code